components/collective-page/sections/Contributions.js

```diff
@@ -364,7 +365,10 @@ const SectionContributions = ({ collective }) => {
         >
           <Grid gridGap={24} gridTemplateColumns={GRID_TEMPLATE_COLUMNS}>
             {(!loading || (isLoadingMore && loading)) &&
-              memberOf?.nodes.map(membership => (
+              uniqWith(
+                memberOf?.nodes,
+                (member1, member2) => member1.role === member2.role && member1?.account.id === member2?.account.id,
+              ).map(membership => (
```

Comment:
Since we're talking about the exact same role and account id, I think this should be fixed on the API side.

Reply:
@kewitz : But when we are returning the `memberOf` from API we also return the `tiers`. So a member can have both `member.account.id` and `role` same but different `tier`.